src/modules/data.js

```diff
@@ -1,28 +1,11 @@
-const table = [
-  {
-    name: 'Mariana',
-    score: 12345,
-  },
-  {
-    name: 'Fernando',
-    score: 12345,
-  },
-  {
-    name: 'Daniela',
-    score: 12345,
-  },
-  {
-    name: 'Julio',
-    score: 12345,
-  },
-  {
-    name: 'Viviana',
-    score: 12345,
-  },
-  {
-    name: 'Marcos',
-    score: 12345,
-  },
-];
+import buildTable from './display.js';
 
-export default table;
\ No newline at end of file
+const URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/sN1XA4iUm9JzCAVPL1P5/scores/';
+
+const displayScores = () => {
+  fetch(URL)
+    .then((response) => response.json())
+    .then((json) => buildTable(json.result));
+};
```

Comment:
- [ ] Please make this function asynchronous also. It will make your code cleaner and better check for response from the fetch.